app/builders/digest_email_builder.rb

```diff
@@ -34,15 +34,7 @@ def body
 
       ---
 
-      # Why am I getting this email?
-
-      #{I18n.t("emails.digests.#{subscription.frequency}.footer_explanation")}
-
-      #{subscriber_list.title}
-
-      [Unsubscribe](#{unsubscribe_url})
-
-      [#{I18n.t!('emails.digests.footer_manage')}](#{PublicUrls.authenticate_url(address: subscriber.address)})
+      #{FooterPresenter.call(subscriber, subscription).strip}
```

Comment:
Could the strip be done in the presenter? It feels a strange extra detail for the caller to consider.

Reply:
Sure, I've amended the commit to do that. I was trying to be consistent with the presenters for content changes and messages, which inject a newline at the end. This was quite surprising, though, so I've added another commit to make them all consistent i.e. to not include a newline at the end.